Microsoft.Toolkit.Uwp.UI/Converters/TypeToObjectConverter.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Data;
+
+namespace Microsoft.Toolkit.Uwp.UI.Converters
+{
+    /// <summary>
+    /// This class returns an object or another, depending on whether the type of the provided value matches another provided Type.
+    /// </summary>
+    public class TypeToObjectConverter : DependencyObject, IValueConverter
```

Comment:
>TypeToObjectConverter [](start = 17, length = 21)

nit: you can seal the clas

Reply:
I'll make this change as part of my license update too.